tasks/linters/hadolint/0.1/hadolint.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: hadolint
+spec:
+  description: |
+    The Hadolint task runs the Hadolint (Haskell Dockerfile Linter) Docker linting tool on Dockerfiles/Containerfiles
+    contained in the repository.
+  params:
+  - name: git-url
+    description: The Git URL from which the test pipeline is originating. This can be from a fork or the original repository.
+    type: string
+  - name: git-revision
+    description: The Git revision (commit SHA) from which the test pipeline is originating.
+    type: string
+  - name: dockerfile-path
+    description: Dockerfile path.
+    type: string
+    default: './Dockerfile'
+  - name: ignore-rules
+    description: Hadolint rules which will be ignored during linting.
+    type: string
+    default: ''
+  - name: failure-threshold
+    description: Rule severity threshold for pipeline failure. One of (error, warning, info, style, ignore)
+    type: string
+    default: 'info'
+  - name: output-format
+    description: >-
+      The output format for the results (tty, json, checkstyle, codeclimate, gitlab_codeclimate, codacy).
+    type: string
+    default: tty
+  steps:
+  - name: clone-refs
+    image: quay.io/konflux-qe-incubator/konflux-qe-tools:latest
```

Comment:
Can use latest of this one: https://github.com/konflux-ci/tekton-integration-catalog/blob/main/.tekton/tekton-integration-catalog-utils-push.yaml#L27